site/public/js/ta-grading-mark.js

```diff
@@ -447,16 +454,32 @@ function calculateMarksPoints(question_num) {
 
     current_row = $('#mark_custom_id-'+question_num);
     var custom_points = parseFloat(current_row.find('input[name=mark_points_custom_'+question_num+']').val());
-    if (isNaN(custom_points)) {
-        current_points += 0;
-    } else {
-        if(custom_points!=0){
-            current_points += custom_points;
-            any_selected = true;
+    var custom_message = current_row.find('textarea[name=mark_text_custom_'+question_num+']').val();
+    if(custom_message == ""){
+        $('#mark_points_custom-' + question_num)[0].disabled=true;
+        $('#mark_points_custom-' + question_num)[0].style.cursor="not-allowed";
+        $('#mark_icon_custom-' + question_num)[0].style.cursor="not-allowed";
+        $('#mark_points_custom-' + question_num)[0].value="";
+    }
+    else{
+        $('#mark_points_custom-' + question_num)[0].disabled=false;
+        $('#mark_points_custom-' + question_num)[0].style.cursor="default";
+        $('#mark_icon_custom-' + question_num)[0].style.cursor="pointer";
+        if($('#mark_points_custom-' + question_num)[0].value==""){
+            $('#mark_points_custom-' + question_num)[0].value="0";
+        }
+        if (isNaN(custom_points)) {
+            current_points += 0;
+        } 
+        else {
+            if(custom_points!=0){
+                current_points += custom_points;
+                any_selected = true;
+            }
```

Comment:
Selecting only a custom mark with zero points makes it look like you have graded the assignment but shows the score as not graded: ![](https://i.imgur.com/ACk1In3.jpg)